### PR TITLE
Fix error logs when running mysql.sql script in identity/dbscripts

### PR DESCRIPTION
**Pull request by Shenali-SJ**
### Purpose
This resolves error logs generated when running mysql script in identity/dbscripts.

### Related Issues
https://github.com/wso2/product-is/issues/16160

### Followup Actions
Bump the carbon-identity-framework version in product-is.

### CODE REVIEW

**Review by Kanapriya — APPROVED** [review]

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql [hunk 1]
```diff
@@ -1,7 +1,7 @@
 CREATE TABLE IF NOT EXISTS IDN_BASE_TABLE (
             PRODUCT_NAME VARCHAR(20),
             PRIMARY KEY (PRODUCT_NAME)
-)ENGINE INNODB;
+)DEFAULT CHARACTER SET latin1 ENGINE INNODB;
```
Comment (Kanapriya) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql:4]: As this is a known limitation for MySQL 8 and in the [documentation](https://is.docs.wso2.com/en/6.0.0/deploy/environment-compatibility/#tested-dbmss) itself, explicitly mentioned that, databases needs to be created with `latin1`. Hence IMO, we don't need to add this in the dbscripts @DMHP WDYT?

BTW, Can you check whether this syntax errors are occurred in bit lower version other than the MYSQL8?

Comment (Kanapriya) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql:4]: As this is a known limitation for MySQL 8 and in the [documentation](https://is.docs.wso2.com/en/6.0.0/deploy/environment-compatibility/#tested-dbmss) itself, explicitly mentioned that, databases needs to be created with `latin1`. Hence IMO, we don't need to add this in the dbscripts @DMHP WDYT?

**Review by Kanapriya — CHANGES_REQUESTED** [review]

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql (hunk 1 above)
Comment (Shenali-SJ) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql:4]: I have tested this using 5.7.12 MySQL Community Server (GPL) and it did not produce any errors.

**Comment by Shenali-SJ** [comment]
It was concluded that using the latin1 charset does not cause any performance issues. A comprehensive analysis of the performance implications of utilizing latin1 has been included in [this document](https://docs.google.com/document/d/1r2Zfy_Dq39KAIjJcbECSkK-0lApfyQfx96JxQS_Md9A/edit?usp=sharing).

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json [hunk 2]
```diff
@@ -297,7 +297,7 @@
   "scim2.consider_total_records_for_total_results_of_ldap": false,
 
   "identity_mgt.recovery.notification.manage_internally": true,
-  "identity_mgt.recovery.callback_url": "[${carbon.protocol}://${carbon.host}:${carbon.management.port}].*[/authenticationendpoint/login.do]*",
+  "identity_mgt.recovery.callback_url": "${carbon.protocol}:\\/\\/${carbon.host}:${carbon.management.port}\\/.*",
```
Comment (DMHP) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json:300]: Is this change intended for this PR?

**Comment by DMHP** [comment]
Please exclude the unintended modification and merge this, as you have tested this with lower versions too.

**Review by Kanapriya — APPROVED** [review]
